### PR TITLE
tcpdump: add page

### PR DESCRIPTION
**Pull request by tcpdump-examples**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by Yutyo — APPROVED** [review]
I think it looks OK at the moment. LGTM!

File: pages/linux/tcpdump.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+`tcpdump -i any port 8000`
+
+- Write packets to a file:
+
+`tcpdump -c 5 -w network_file_linux.pcap -i any`
+
+- Capture packets from a specific protocol:
+
+`tcpdump -i ens160 -c 5 -nn tcp`
```
Comment (bl-ue) [inline pages/linux/tcpdump.md:23]: Why are there two `n`'s here?
```suggestion
`tcpdump --interface={{interface}} --count={{5}} -nn tcp`
```

File: pages/linux/tcpdump.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Filter tcpdump packets from specific source & dest host:
+
+`tcpdump src 100.10.8.121`
+
+- Rotate tcpdump packets:
+
+`$ tcpdump -i ens160 -w /tmp/network-%H-%M.pcap -W 48 -G 300 -C 100`
```
Comment (bl-ue) [inline pages/linux/tcpdump.md:31]:
```suggestion
`tcpdump --interface={{interface}} -w {{path/to/file.pcap}} -W {{48}} -G {{300}} -C {{100}}`
```

File: pages/linux/tcpdump.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+`tcpdump -c 5 -w network_file_linux.pcap -i any`
+
+- Capture packets from a specific protocol:
+
+`tcpdump -i ens160 -c 5 -nn tcp`
+
+- Filter tcpdump packets from specific source & dest host:
+
+`tcpdump src 100.10.8.121`
```
Comment (bl-ue) [inline pages/linux/tcpdump.md:27]:
```suggestion
`tcpdump {{100.10.8.121}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hey there @tcpdump-examples!  This is long-needed page, so I'd like to thank you for it!

However, it isn't consistent with the rest of our pages in a few spots, so please apply my syntactic suggestions below so I can have a look at the semantics 😉

**Review by sbrl — COMMENTED** [review]
Looks ok to me when @bl-ue's comments are addressed :-)

File: pages/linux/tcpdump.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Tcpdump
+
+> [useful tcpdump examples](https://www.howtouselinux.com/post/10-useful-tcpdump-command-examples).
```
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/linux/tcpdump.md:3]: ping @tcpdump-examples

File: pages/linux/tcpdump.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# tcpdump
+
+> [useful tcpdump examples](https://www.howtouselinux.com/post/10-useful-tcpdump-command-examples).
+
+- Capture traffic on specific interface:
```
Comment (marchersimon) [inline pages/linux/tcpdump.md:5]:
```suggestion
- Capture traffic on a specific interface:
```

File: pages/linux/tcpdump.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/linux/tcpdump.md:3]: According to the maintainers guide we can commit the chnanges ourselves and merge, right?

[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/linux/tcpdump.md:3]: Yeah, it's been almost a month now so I think we should just take over 👍🏻

**Review by marchersimon — APPROVED** [review]

**Comment by bl-ue** [comment]
Haha, I just happened to notice that we (me, @tcpdump-examples, @sbrl, @marchersimon and @Yutyo) all missed the fact that [`common/tcpdump.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/tcpdump.md) (as opposed to this PR's _[`common/linux.md`](https://github.com/tldr-pages/tldr/pull/5340/files#diff-37ab68d030f793f4a18b75a2eb21caf0d126ac5b841ebd2b9eb91c287376519c)_) is one of our oldest pages, being added in PR #138. :smile:

I'm going to close this PR as the existing page already 8 examples and the author of this PR appears to have abandoned it.

**Pull request closed without merging**